src/ui/public/vis/editors/components/editor_options_group.js

```diff
@@ -18,10 +20,10 @@ import {
  * to produce an aligned look and feel.
  */
 function EditorOptionsGroup(props) {
-  return (
+  const collapsibleOptionGroup = (
     <EuiPanel
       grow={false}
-      className="editorOptionsGroup__panel"
+      className={`editorOptionsGroup__panel ${props.className}`}
```

Comment:
This would write `editorOptionsGroup__panel undefined` to the class if the property hasn't been specified. We usually use [classnames](https://github.com/JedWatson/classnames) for concatenating (and conditionally) apply classnames.

```js
import className from 'classnames';

const panelClass = className('editorOptionsGroup__panel', props.className);

return <EuiPanel className={panelClass} />;
```